modules/apps/portal-odata/portal-odata-api/src/main/java/com/liferay/portal/odata/entity/StringEntityField.java

```diff
@@ -44,4 +44,25 @@ public StringEntityField(
 			filterableAndSortableFieldNameFunction, String::valueOf);
 	}
 
+	/**
+	 * Creates a new {@code StringEntityField} with a {@code Function} to
+	 * convert the entity field's name to a filterable/sortable field name for a
+	 * locale.
+	 *
+	 * @param  name the entity field's name
+	 * @param  sortableFieldNameFunction the sortable field name {@code
+	 *         Function}
+	 * @param  filterableFieldNameFunction the filterable field name {@code
+	 *         Function}
+	 * @review
+	 */
+	public StringEntityField(
+		String name, Function<Locale, String> sortableFieldNameFunction,
+		Function<Locale, String> filterableFieldNameFunction) {
```

Comment:
Might want to alphabetize the function names.

Reply:
(I hope the IDE Refactor takes care of all instances)

Reply:
The reason I did that order was to match the parent class: https://github.com/liferay-search/liferay-portal/blob/bc89c7fb837632c72cc7b7754d93eff2ca846085/modules/apps/portal-odata/portal-odata-api/src/main/java/com/liferay/portal/odata/entity/EntityField.java#L46-L49

Also, SF didn't complain haha. Do you know if the preference would be to alphabetize the function names regardless?